lib/rules/block-opening-brace-newline-after/index.js

```diff
@@ -43,8 +42,25 @@ const rule = function(expectation) {
         return;
       }
 
+      // next node with checking newlines after comment
+      function nextNode(startNode) {
+        if (!startNode || !startNode.next) return null;
+
+        if (startNode.type === "comment") {
+          const newLines = ["\n", "\r\n", "\n ", "\r\n "];
```

Comment:
Something wrong about this. Maybe use regexp instead of an array and `indexOf`:

```js
const reNewLine = /\r?\n/;

const index = reNewLine.test(startNode.raws.before);
```